WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Posts/PostsCardViewModel.swift

```diff
@@ -291,6 +251,11 @@ extension PostsCardViewModel: NSFetchedResultsControllerDelegate {
         guard let dataSource = viewController?.tableView.dataSource as? DataSource else {
             return
         }
+
+        guard updatePostsInfoIfNeeded() == false else {
```

Comment:
I think this part here might be causing the deleted post to still show after being deleted.

Reply:
This part basically detects when the fetchController has zero posts. When this is detected, we halt reloading the posts list table view and trigger the dashboard collection to reload all cards.